src/ASM/LR/ASMu3D.h

```diff
@@ -52,7 +52,9 @@ class ASMu3D : public ASMLRSpline, public ASM3D
     //! \param pch Patch the cache is for
     //! \param plcy Cache policy to use
     //! \param b Basis to use
-    BasisFunctionCache(const ASMu3D& pch, ASM::CachePolicy plcy, int b);
+    //! \param useBezier True to to use bezier extraction
```

Comment:
A little studder in the doxy here (to to)

Reply:
fixed.